### ISSUE TITLE
Errors build in Xcode 13

### ISSUE THREAD

**Issue by asepms92** [CloverHackyColor/CloverBootloader#453]
Hi @SergeySlice Thanks for hard work, i got issues build/compile latest [commits](https://github.com/CloverHackyColor/CloverBootloader/commit/83b93234bec29dd889714a85825e482f16cc2caf) , trying with buildme/ebuild.sh same as results.. tested with Xcode 13 (13A233) and GCC (10.2.0)
![Screen Shot 2021-09-29 at 22 27 10](https://user-images.githubusercontent.com/43111808/135310607-e3eee644-cc45-482b-8f9a-75746f04ff1d.png)

**Comment by SergeySlice** [comment]
Yes, there is wrong indentation in this part of codes. But C, C++ is not Python.Indentation doesn't matter.
Logically all is good here.
You may set -Wno-misleading-indentation to avoid such errors or reformat all 1000 files of sources.
